test/e2e/tests/bridge/bridge-button-opens-portfolio.spec.ts

```diff
@@ -17,21 +17,20 @@ describe('Click bridge button @no-mmi', function (this: Suite) {
 
   it('loads portfolio tab from asset overview when flag is turned off', async function () {
     await withFixtures(
-      // withErc20 param is false, as we test it manually below
```

Comment:
we support passing a chainId to the `withTokensControllerERC20` method, so now we are able to use fixtures and remove all extra steps